test/tcp/TcpServer.cpp

```diff
@@ -62,6 +62,11 @@ class ServerRequestHandler : public RequestHandler {
     response.onNext(folly::IOBuf::copyBuffer("from server2"));
     response.onComplete();
   }
+
+  void handleFireAndForgetRequest(Payload request) override {
+    std::cout << "ServerRequestHandler.handleFireAndForgetRequest "
```

Comment:
This is Yuri's stuff, but I'd prefer to use logger for everything. AFAIK, logger is not synchronised with stdout, so this might result in misleading output.